resources/views/internlink/policy.blade.php

```diff
@@ -0,0 +1,21 @@
+@extends('layouts.panel')
+@section('panel-cols', '10')
+@section('panel-offset', '1')
+@section('panel-heading', 'InternLink')
+@section('panel-body')
+    <h2>Usage Policy</h2>
+    <h4>InternLink is meant to be a tool for current CUMSA members (Users) to seek the advice of past/current members (Contributors)
+        with regards to internship and job applications. Below are the privacy policy for the platform.</h4>
+    <h3><u>For contributors</u></h3>
+    <h4>CUMSA will store the information you have provided on a server and make it available only to members of CUMSA (by authentication via RAVEN).
+        By providing us with these information, you assent to being contacted by current and recently graduated CUMSA members for the provision of general and specific advice about your current job and any past jobs/internships you may have performed.
```

Comment:
Grammar nazi
* these or this?
* I'm not sure assent is used that way (try googling "assent to be contacted"). Seems like you can either agree to be contacted, or you give assent to be contacted.